services/explorer/src/utils/TransactionRow.tsx

```diff
@@ -6,6 +6,13 @@ import { HashLink } from '../components/Links';
 import { BlockData, TransactionTableData, TransactionStatus } from '../models';
 import { HashLinkType } from './Enums';
 
+export const toHexStatus = (status: number): TransactionStatus =>
```

Comment:
You don't need this. We already have the formatTransactionStatus method in utils. If needed, update that method and remove this one. But make sure it works the same where it is used.